src/transformers/models/blip/modeling_blip.py

```diff
@@ -795,6 +795,12 @@ def __init__(self, config: BlipConfig):
         # Initialize weights and apply final processing
         self.post_init()
 
+    def get_input_embeddings(self):
+        return self.text_model.get_input_embeddings()
+
+    def set_input_embeddings(self, value):
+        self.text_model.set_input_embeddings(value)
```

Comment:
hmm, i see that in PreTrainedModel we try to get the method from `base_model` and prob we can fallback to that by indicating the `base_model_prefix`

I am not very sure yet how the prefix is used when loading the model, so lemme quick check that state dict is still correctly loaded

Reply:
Update: yes the idea works and loading happens same way as without the `base_model_prefix`. But some of the tests will fail because of the composite nature of `BlipConfig` (`test_correct_missing_keys`)

I will take this noted, and will add it to my TODO list. But I believe it would force us to refactor `from_pretrained` to work well with composite models

Reply:
Okay